src/typed-method-types/workflows/triggers/base_response.ts

```diff
@@ -22,9 +22,16 @@ export type BaseTriggerResponse<
       /** @default "workflow" */
       type: string;
       /** @description the workflow's input parameters */
-      input_parameters: WorkflowDefinition["input_parameters"];
+      input_parameters: {
+        type: string;
+        name: string;
+        title: string;
+        description?: string;
+        is_required: boolean;
+      }[];
```

Comment:
I think for now your suggestion makes sense.
One confusing aspect here is that these are the webapp-modified versions of the parameter descriptions (e.g. they have `is_required` rather than the developer-facing `required` array field on the entirety of a parameter set). 😣